launcher/net/MetaCacheSink.cpp

```diff
@@ -88,6 +93,37 @@ Task::State MetaCacheSink::finalizeCache(QNetworkReply & reply)
     }
 
     m_entry->setLocalChangedTimestamp(output_file_info.lastModified().toUTC().toMSecsSinceEpoch());
+
+    { // Cache lifetime
+        if (reply.hasRawHeader("Cache-Control")) {
+            auto cache_control_header = reply.rawHeader("Cache-Control");
+            // qDebug() << "[MetaCache] Parsing 'Cache-Control' header with" << cache_control_header;
```

Comment:
Did you mean to leave these in?

Reply:
Yes! While I don't think it's worth leaving those on a build, hence the commenting, I do think they are helpful with debugging this part of the code should the necessity arise. So I formatted it nicely and left it as a comment to our future debuggers :)
do you think this needs to be removed? :pleading_face: /gen

Reply:
No, that's fair.